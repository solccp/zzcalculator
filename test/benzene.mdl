C6H6 Benzene 71432
##CCCBDB 1281100:47
Geometry Optimized at B3LYP/cc-pVTZ
 12 12  0  0  0  0  0  0  0  0    V2000
    0.0000    1.3907    0.0000 C  0000000000000000000
    1.2044    0.6954    0.0000 C  0000000000000000000
    1.2044   -0.6954    0.0000 C  0000000000000000000
    0.0000   -1.3907    0.0000 C  0000000000000000000
   -1.2044   -0.6954    0.0000 C  0000000000000000000
   -1.2044    0.6954    0.0000 C  0000000000000000000
    0.0000    2.4728    0.0000 H  0000000000000000000
    2.1415    1.2364    0.0000 H  0000000000000000000
    2.1415   -1.2364    0.0000 H  0000000000000000000
    0.0000   -2.4728    0.0000 H  0000000000000000000
   -2.1415   -1.2364    0.0000 H  0000000000000000000
   -2.1415    1.2364    0.0000 H  0000000000000000000
  1  2  4  0     0  0
  1  6  4  0     0  0
  1  7  1  0     0  0
  2  3  4  0     0  0
  2  8  1  0     0  0
  3  4  4  0     0  0
  3  9  1  0     0  0
  4  5  4  0     0  0
  4 10  1  0     0  0
  5  6  4  0     0  0
  5 11  1  0     0  0
  6 12  1  0     0  0
