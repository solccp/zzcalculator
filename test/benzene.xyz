12
C6H6 Benzene 71432
C          0.00000        1.39070        0.00000
C          1.20440        0.69540        0.00000
C          1.20440       -0.69540        0.00000
C          0.00000       -1.39070        0.00000
C         -1.20440       -0.69540        0.00000
C         -1.20440        0.69540        0.00000
H          0.00000        2.47280        0.00000
H          2.14150        1.23640        0.00000
H          2.14150       -1.23640        0.00000
H          0.00000       -2.47280        0.00000
H         -2.14150       -1.23640        0.00000
H         -2.14150        1.23640        0.00000
