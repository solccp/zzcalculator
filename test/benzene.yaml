---
structure_id: 1
number_of_atoms: 6
double_bonds:
- [1, 6]
- [5, 4]
- [3, 2]
---
structure_id: 2
number_of_atoms: 6
double_bonds:
- [1, 2]
- [3, 4]
- [6, 5]
---
structure_id: 3
number_of_atoms: 6
rings:
- [1, 2, 3, 4, 5, 6]
