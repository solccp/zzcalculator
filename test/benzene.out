 2 + 1 x
 total: 3
